ironfish/src/rpc/clients/socketClient.ts

```diff
@@ -34,10 +35,21 @@ export abstract class RpcSocketClient extends RpcClient {
   abstract client: IpcClient | net.Socket | null
   abstract isConnected: boolean
   abstract connection: Partial<RpcClientConnectionInfo>
+  rpcAuthToken: string
```

Comment:
nit - `authToken` since it's a property of `RpcSocketClient`?

Reply:
This should be nullable since it's not required in a client.